lib/ddtrace/contrib/active_record/patcher.rb

```diff
@@ -95,6 +97,22 @@ def self.sql(_name, start, finish, _id, payload)
         rescue StandardError => e
           Datadog::Tracer.log.error(e.message)
         end
+
+        def self.instantiation(_name, start, finish, _id, payload)
+          span = get_option(:tracer).trace(
+            'active_record.instantiation',
+            resource: payload.fetch(:class_name),
+            service: get_option(:service_name),
+            span_type: Datadog::Ext::Ruby::TYPE
+          )
+
+          span.set_tag('active_record.instantiation.class_name', payload.fetch(:class_name))
+          span.set_tag('active_record.instantiation.record_count', payload.fetch(:record_count))
+          span.start_time = start
+          span.finish(finish)
+        rescue StandardError => e
+          Datadog::Tracer.log.error(e.message)
```

Comment:
not sure.... these `rescue` are in a critical path and can spam users' logs. What about making all of them `log.debug`?

Reply:
This is what Rails was already doing for `sql`. If we change it here, we should probably change it there, too.

Reply:
Yes I meant all occurrences in this integration (Rails included). I'm worried that for an internal problem we may cause issues like it happened before.

Reply:
Alright, I can do that then.